src/components/card/card.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+import styles from './card.module.scss';
+
+interface Props {
+  iconUrl: string;
+  children: string;
+  href: string;
+  altTextForIcon: string;
+}
+
+export const Card = ({
+  children, iconUrl, href, altTextForIcon,
+}: Props) => (
+  <a href={href} className={styles.card}>
+    <img src={useBaseUrl(iconUrl)} alt={altTextForIcon} />
```

Comment:
Do not use `<img>` tag for things that are not **_content_**
Consider implementing icons via any alternative tag (`<i>`, `<span>` ) and use `background-image` css property